compass/ocean/tests/overflow/rpe_test/analysis.py

```diff
@@ -0,0 +1,188 @@
+import numpy as np
+import xarray
+import matplotlib.pyplot as plt
+import cmocean
+
+from compass.step import Step
+from compass.ocean.rpe import compute_rpe
+
+
+class Analysis(Step):
+    """
+    A step for plotting the results of a series of RPE runs in the overflow
+    test group
+
+    Attributes
+    ----------
+    resolution : str
+        The resolution of the test case
+
+    nus : list of float
+        A list of viscosities
+    """
+    def __init__(self, test_case, resolution, nus):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.TestCase
+            The test case this step belongs to
+
+        resolution : str
+            The resolution of the test case
+
+        nus : list of float
+            A list of viscosities
+        """
+        super().__init__(test_case=test_case, name='analysis')
+        self.resolution = resolution
+        self.nus = nus
+
+        self.add_input_file(
+            filename='initial_state.nc',
+            target='../initial_state/ocean.nc')
+
+        for index, nu in enumerate(nus):
+            self.add_input_file(
+                filename='output_{}.nc'.format(index+1),
+                target='../rpe_test_{}_nu_{}/output.nc'.format(index+1, nu))
+
+        self.add_output_file(
+            filename='sections_overflow_{}.png'.format(resolution))
+        self.add_output_file(filename='rpe_t.png')
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+        section = self.config['overflow']
+        nx = section.getint('nx')
+        ny = section.getint('ny') - 2
+        rpe = compute_rpe()
+        _plot(nx, ny, self.outputs[0], self.nus, rpe)
+
+
+def _plot(nx, ny, filename, nus, rpe):
```

Comment:
`nx` and `ny` are not used so should be removed.

Reply:
Resolved by 2c1147cc